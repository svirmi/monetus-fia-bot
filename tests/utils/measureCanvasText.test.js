const measureCanvasText = require('../../lib/utils/measureCanvasText')

test('utils.measureCanvasText', () => {
  const text = `┌────────┬───────────────┬────────┬────────┐
│ Ticker │ Asset         │      % │      Δ │
├────────┼───────────────┼────────┼────────┤
│ LEET1  │ 1337! Corp.   │ 50.10% │ +0.13% │
│ OCTO3  │ Octocat Corp. │ 49.90% │ -0.05% │
├────────┼───────────────┼────────┼────────┤
│ TOTAL  │               │   100% │ +0.08% │
└────────┴───────────────┴────────┴────────┘`

  const fontFamily = 'Fira Code'
  const fontSize = '20px'

  expect(measureCanvasText(text, fontFamily, fontSize)).toMatchSnapshot()
})
