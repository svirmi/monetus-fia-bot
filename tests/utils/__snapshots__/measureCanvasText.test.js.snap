// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils.measureCanvasText 1`] = `
Object {
  "height": 192,
  "numberOfLines": 8,
  "width": 528,
}
`;
